database/queries/add_career.js

```diff
@@ -3,8 +3,7 @@ const dbConnection = require('../db_connection');
 const addCareer = (data, cb) => {
   const newCareerQuery = `INSERT INTO careers (
     title_ar, title_he, tagline_ar, tagline_he,
-    description_ar, description_he, image_url,
-    salary_start, salary_ten_year, icon_url)
+    description_ar, description_he, salary_start, salary_ten_year, icon_name, icon_color)
```

Comment:
Not sure about the inconsistent line size on this query. Format for readability.